ApiBundle/Transformer/ApiClientCollectionTransformer.php

```diff
@@ -13,15 +13,15 @@
 class ApiClientCollectionTransformer extends AbstractTransformer
 {
     /**
-     * @param Collection $mixed
+     * @param ArrayCollection $apiClientCollection
      *
      * @return FacadeInterface
      */
-    public function transform($mixed)
+    public function transform($apiClientCollection)
```

Comment:
you should let $mixed as the parameter name, this way it will be consistent with the interface